apps/openmw/mwrender/renderingmanager.cpp

```diff
@@ -74,10 +75,54 @@
 #include "recastmesh.hpp"
 #include "fogmanager.hpp"
 #include "objectpaging.hpp"
-
+#include "groundcover.hpp"
 
 namespace MWRender
 {
+    class GroundcoverUpdater : public SceneUtil::StateSetUpdater
```

Comment:
Would this class not be better off defined in one of the groundcover-related files? `renderingmanager.cpp` is pretty big already.